lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -21,6 +21,13 @@ all_interfaces:
   get_context: ~
   get_value: '/^interface (.*)/'
 
+bfd_echo:
+  _exclude: [ios_xr]
+  kind: boolean
+  get_value: '/^(?:no )?bfd echo$/'
+  set_value: '<state> bfd echo'
+  default_value: true
```

Comment:
@chrisvanheuveln Not I understand the concern.  Wouldn't the same also apply for all BGP properties if `feature bgp` is enabled/disabled?